static/src/javascripts/projects/commercial/commercial-metrics.ts

```diff
@@ -1,41 +1,32 @@
-import { sendCommercialMetrics } from '@guardian/commercial-core';
+import {
+	initCommercialMetrics,
+	bypassCommercialMetricsSampling as switchOffSampling,
```

Comment:
Good call to use an import alias until we can rename this function in commercial-core. 
Personally I'm not sure that this name makes it more clear that by calling the function the sampling rate for commercial metrics will be set to 100%. I think "switching off sampling" has the same problem as "bypassing sampling", which is that it could be interpreted as setting the sampling rate to 0%.
What about something like `setSamplingTo100Percent`?

Reply:
Yep agree it was my suggestion but let's come up with better naming that makes sense to everyone.

Reply:
This is a good point. I'll make a note to consider the name in the trello card for this.